Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -0,0 +1,51 @@
+Import-VstsLocStrings -LiteralPath $PSScriptRoot/module.json
+
+function Add-Tls12InSession {
+    [CmdletBinding()]
+    param()
+
+    try {
+        if ([Net.ServicePointManager]::SecurityProtocol -notcontains 'Tls12') {
+            [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]3072
```

Comment:
wouldn't type-casing fail if enum does not contain 3072. you can just assign 3072 directly.

Reply:
I think we are ok here. Typecasing works for .Net 4.0.
For .Net 3.5 and below, TLS 1.2 is not natively supported. So adding 3072 would also fail there as this number would not correspond to any known protocol. To add TLS 1.2 there, you would need to install a patch released by Microsoft. That patch adds TLS12 to enum. So typecasing would work there also.